src/apprt/gtk/version.zig

```diff
@@ -39,3 +39,20 @@ pub inline fn atLeast(
 
     return false;
 }
+
+test "atLeast" {
+    const std = @import("std");
+    const testing = std.testing;
+
+    try testing.expectEqual(true, atLeast(c.GTK_MAJOR_VERSION, c.GTK_MINOR_VERSION, c.GTK_MICRO_VERSION));
```

Comment:
If you're checking a boolean, you can do:
```
try testing.expect(atLeast(c.GTK_MAJOR_VERSION, c.GTK_MINOR_VERSION, c.GTK_MICRO_VERSION);
try testing.expect(!atLeast(c.GTK_MAJOR_VERSION + 1, c.GTK_MINOR_VERSION, c.GTK_MICRO_VERSION);
```

Reply:
Thanks for the suggestion, I've updated accordingly.